.github/workflows/build.yaml

```diff
@@ -60,16 +60,18 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v5
 
-      - name: Install Winget
-        if: matrix.runner.name == 'windows-11-arm'
-        uses: ./.github/actions/install-winget
-        with:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Install Winget PowerShell Module
         shell: pwsh
         run: Install-Module -Name Microsoft.WinGet.Client -Repository PSGallery -Force
 
+      - name: Install Winget
+        if: matrix.runner.name == 'windows-11-arm'
```

Comment:
Maybe add a brief comment explaining why we need this extra step for ARM

Reply:
addressed